test/unit/cms/s3_test.rb

```diff
@@ -64,4 +64,16 @@ class CMS::S3Test < ActiveSupport::TestCase
     assert CMS::S3.options[:endpoint]
     assert_not CMS::S3.options[:use_fips_endpoint]
   end
+
+  test "#credentials returns nil when not CMS::S3 is not enabled" do
```

Comment:
redentials returns nil when CMS::S3 is not enabled would be better

Reply:
Much better indeed. It even makes sense now 😅 
Updated in 386b70e.